src/Model/SnapshotInterface.php

```diff
@@ -17,6 +17,13 @@
  * Defines methods to interact with the persistency layer of a SnapshotInterface.
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * @method int|null           getTargetId()
```

Comment:
Are these methods intended to be part of the interface in a next major release?

Reply:
We should add all @methods in the next major version